sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,282 @@
+- Added experimental `set_kernel_properties` API and `use_double_grf` property
+  for ESIMD. [9a55da53]
+- Added "eager initialization" mode to Level Zero PI plugin. [c1459598]
+- Added full support of element wise operations on `joint_matrix` on CUDA
+  backend including `bfloat16` support. [0a1d751b]
+
+### Documentation
+
+- Added stateful to stateless mem access conversion
+  [design document](sycl/doc/design/ESIMDStatelesAccessors.md). [3e03f300]
+- Added [`sycl_ext_oneapi_complex`](sycl/doc/extensions/proposed/sycl_ext_oneapi_complex.asciidoc)
+  extension proposal. [01589da5]
+- Updated [`sycl_ext_intel_fpga_device_selector`](sycl/doc/extensions/supported/sycl_ext_intel_fpga_device_selector.asciidoc)
+  extension to add `fpga_simulator_accessor`. [9bef890d]
+- Added [`sycl_ext_intel_fpga_kernel_interface_properties`](sycl/doc/extension/proposed/sycl_ext_intel_fpga_kernel_interface_properties.asciidoc) extension proposal. [4b6bd14b]
+- Updated [`sycl_ext_oneapi_complex_algorithms`](sycl/doc/extensions/proposed/sycl_ext_oneapi_complex_algorithms.asciidoc)
+  extension to include `sycl::complex` as supported type for algorithms. [07c5b48f]
+- Clarified sub-group size calculation in [`sycl_ext_oneapi_invoke_simd`](sycl/doc/extensions/experimental/sycl_ext_oneapi_invoke_simd.asciidoc) extension spec. [9b33ad0f]
+- Updated [`sycl_ext_oneapi_accessor_properties`](sycl/doc/extensions/supported/sycl_ext_oneapi_accessor_properties.asciidoc)
+  to mark `has_property` API as `noexcept`. [7805aa3f]
+- Updated [`sycl_ext_intel_device_info`](sycl/doc/extensions/supported/sycl_ext_intel_device_info.md)
+  to support querying free device memory. [0eeef2b3]
+- Updated [`sycl_ext_oneapi_matrix`](sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix.asciidoc)
+  with description of new matrix features. [770f540d]
+- Moved [`sycl_ext_oneapi_invoke_simd`](sycl/doc/extensions/experimental/sycl_ext_oneapi_invoke_simd.asciidoc)
+  extensions specification from `proposed` to `experimental` because
+  implementation is available. [6bee3440]
+
+## Improvements
+
+### SYCL Library
+
+- Ensured that a correct errc thrown for unassociated placeholder accessor (4.7.6.9) [4f9935ad]
+- Removed dependency on OpenCL ICD Loader from the runtime [90e8b5ef]
+- Added `group::get_linear_id(int)` method [6e83c127]
+- Added support for `ZEBIN` format to persistent caching mechanism [34dcf83d]
+- Added identification mechanism for binaries in newer `ZEBIN` format. [f4dee549]
+- Fixed uses of common macro names in the imlementation's header files [e87adfd2]
+- Switched to use `struct` information descriptors in accordance with SYCL 2020.
+  Removed some deprecated infromation queries. [b3cbda57]
+- Updated `kernel_device_specific::max_sub_group_size` query to match SYCL 2020
+  spec. Deprecated the old variant. [7842d056]
+- Deprecated SYCL 1.2.1 device selectors. [c058380f]
+- Improved error messages reported for unsupported devie partitioning. [1c9ddbaa]
```

Comment:
```suggestion
- Improved error messages reported for unsupported device partitioning. [1c9ddbaa]
```

Reply:
Applied in ed5e71d464608b7be8fd28ee14249979e67e4fb8